termsaver

```diff
@@ -74,7 +74,7 @@ def usage():
     # Print the main termsaver header
     ScreenBase.usage_header()
 
-    print _("""Usage: %(app_name)s [screen] [options]
```

Comment:
"_()" is a function for internationalization. I noticed you removed all of them, which would break with your changes. Can you check this?

Reply:
I'll check into the internationalization aspect (Not exactly sure what that does!) but in python3 print is a hard function now, so the enclosing parenthesis are no longer optional.

Reply:
Can I clarify if we're discussing https://docs.python.org/3/library/gettext.html ?  It seems that the print( _(<string>) ) syntax is still supported but lots of the functions are labeled with "Deprecated since version 3.8, will be removed in version 3.10."  If true, it's obviously a separate issue, but I figure it's worth noting here.  I'll issue another commit and return the internationalization strings along with a fixed print statement.

Reply:
From the link you provided, only few functions in the module are deprecated, which happens all the time. The module itself is not. 
I can see this work is incompleted (need the translation files to be available, then compiled, etc, which I did not do), but the code is ready for it, so I'd like to maintain this if possible. Just apply the _ function on the top of the new brackets, something like:

`
print _("test")
`
into:
`
print ( _("test") )
`

Reply:
Yep. New commit incoming.